docs/releasenotes/release-1.127.md

```diff
@@ -1,35 +1,49 @@
-# v1.126.0
+* Special shout-outs to @acpana, @anhdle-sso, @cheftako, @ericpang777, @gemmahou, @haiyanmeng, @jasonvigil, @jingyih, @justinsb, @maqiuyujoyce, @nb-goog, @xiaoweim, @yuwenma, @ziyue-101 for their contributions to this release.
 
-** This version is not yet released; this document is gathering release notes for the future release **
+## Announcement 
 
-* ...
 
-* Special shout-outs to ... for their contributions to this release.
-TODO: list contributors with `git log v1.126.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+## New Beta Resources (Direct Reconciler):
 
-## Announcement 
+* [`BigQueryAnalyticsHubListing`](https://cloud.google.com/config-connector/docs/reference/resource-docs/bigqueryanalyticshub/bigqueryanalyticshublisting)
 
-## New Beta Resources (Direct Reconciler):
+* [`FirestoreDatabase`](https://cloud.google.com/config-connector/docs/reference/resource-docs/firestore/firestoredatabase)
+
+* [`WorkstationConfig`](https://cloud.google.com/config-connector/docs/reference/resource-docs/workstations/workstationconfig)
 
-* `WorkstationConfig`
-* `Workstation`
-* `FirestoreDatabase`
+* [`Workstation`](https://cloud.google.com/config-connector/docs/reference/resource-docs/workstations/workstation)
 
 ## New Fields:
 
+* `BigQueryDataTransferConfig`
+
+  * Added `spec.scheduleOptionsV2` to customize the different types of data transfer schedule.
+  * Added `status.observedState.error` with detailed information about reason of the latest config failure.
+
 * `GkeHubFeatureMembership`
-  * Added `spec.configmanagement.management` field to enable Config Sync Auto Upgrade.
 
-* `Placeholder`
+  * Added `spec.configmanagement.management` to enable Config Sync Auto Upgrade. This is an opt-in feature and you need to turn on the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on the object.
+
 
 ## Modified Beta Reconciliation
 
-We migrated the following reconciliation from the TF-based or DCL-based controller to the new Direct controller to enhance the reliability and performance. The resource CRD is unchanged.
+We added the direct controller support for the following 3 resources to enhance the reliability and performance. Add `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on the object to use the direct controller. CRD is backward compatible.
+
+* `GkeHubFeatureMembership`
```

Comment:
GKEHubFeatureMembership is now a pure direct resource and doesn't need the direct annotation afaik.